### PR TITLE
Lab08

### PR DESCRIPTION
**Pull request by MalexSw**
Add Lab8

### CODE REVIEW

File: Lab08/main.c [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+
+
+void calcOfNegativeElementsAverage(int numberOfElements) {
+    int array[numberOfElements];
+    int minValue = -10;
+    int maxValue = 15;
+    int sumOfNegNumbers = 0;
+    int amOfNegativeNum = 0;
+
```
Comment (volnsav) [inline Lab08/main.c:13]: This is the wrong approach, you need to pass the array and its size as arguments.

File: Lab08/main.c [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+
+
+void secondGreatest(int numberOfElements) {
+    int array[numberOfElements];
+    int minValue = -10;
+    int maxValue = 15;
+    int theGreatest = minValue;
+    int secondGreatest = minValue;
```
Comment (volnsav) [inline Lab08/main.c:51]: See the comment above.

File: Lab08/main.c [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+
+
+
+void arrayChanging(int numberOfElements) {
+    int array[numberOfElements];
+    int minValue = -10;
+    int maxValue = 15;
+    char inputOperation = 'a'; // a fo addition, s for subtraction, m for multiplying and d for dividing
+    int valForModification = 3;
```
Comment (volnsav) [inline Lab08/main.c:92]: See the comment above

File: Lab08/main.c [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+
+
+
+
+void sortedArraysMerging(int numberOfElements) {
+    int array1[numberOfElements];
+    int array2[numberOfElements];
+    int minValue = -10;
+    int maxValue = 15;
+    int mergedArray[2 * numberOfElements];
```
Comment (volnsav) [inline Lab08/main.c:154]: See the comment above

File: Lab08/main.c [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+
+
+void evenAndOddNumbersDistributing(int numberOfElements) {
+    int array[numberOfElements];
+    int evenArray[numberOfElements]; 
+    int oddArray[numberOfElements];
+    int minValue = -10;
+    int maxValue = 15;
+    int evenCount = 0;
+    int oddCount = 0;
```
Comment (volnsav) [inline Lab08/main.c:232]: see the comment abobe

**Review by volnsav — CHANGES_REQUESTED** [review]
Please delete the binaries, review and add comments.

**Comment by MalexSw** [comment]
Changes are made

**Pull request closed without merging**